README.adoc

```diff
@@ -185,11 +209,23 @@ Greetings from Spring Boot!
 ----
 ====
 
-== Add Unit Tests
+== Add Tests
 
 You will want to add a test for the endpoint you added, and Spring Test provides some
 machinery for that.
 
+You should see the following dependency in your `build.gradle` file:
```

Comment:
I think we should still keep Maven and Gradle in our guides

Reply:
100%. As I mentioned in my reply to Phil's global comment, our audience uses both (sometimes within the same team). We'd leave some large percentage of the audience behind if we dropped either one.